teuthology/task/install.py

```diff
@@ -1264,6 +1361,11 @@ def task(ctx, config):
     assert isinstance(config, dict), \
         "task install only supports a dictionary for configuration"
 
+    rhbuild=None
+    if config.get('rhbuild'):
+        rhbuild=config.get('rhbuild')
+        log.info("Build is %s " % rhbuild)
```

Comment:
I can't actually find where ``rhbuild`` is used anywhere. What's the intention here?